src/server/game/Scripting/ScriptReloadMgr.cpp

```diff
@@ -67,6 +67,9 @@ namespace fs = boost::filesystem;
 #ifdef _WIN32
     #include <windows.h>
     #define HOTSWAP_PLATFORM_REQUIRES_CACHING
+#elif __APPLE__
+    #include <dlfcn.h>
+    #define HOTSWAP_PLATFORM_REQUIRES_CACHING
```

Comment:
Ah ok, don't know why that is the case, but then just let the system cache the binaries for the OSX platform.